graphblas/operator.py

```diff
@@ -1532,13 +1595,31 @@ def _from_indexunary(cls, iop):
 
     @classmethod
     def register_anonymous(cls, func, name=None, *, parameterized=False, is_udt=False):
+        """
+        Register a SelectOp without registering it in the ``graphblas.select`` namespace.
+
+        Because it is not registered in the namespace, the name is optional.
+        """
         if parameterized:
             return ParameterizedSelectOp(name, func, anonymous=True, is_udt=is_udt)
         iop = IndexUnaryOp._build(name, func, anonymous=True, is_udt=is_udt)
         return SelectOp._from_indexunary(iop)
 
     @classmethod
     def register_new(cls, name, func, *, parameterized=False, is_udt=False, lazy=False):
+        """
+        Register a SelectOp. The name will be used to identify the SelectOp in the
+        ``graphblas.select`` namespace.
+
+        The function will also be registered as a IndexUnaryOp with the same name.
+
+        .. code-block:: python
+
+            >>> from graphblas.operator import SelectOp
```

Comment:
This is interesting though.  Do we need both `.. code-block:: python` and `>>>`?

Reply:
Just tried it out and `>>>` alone will work. No need for additional code-block notation.